### DISCUSSION TITLE
Need to use custom env files for accessing environment variables in schema.prisma datasource.

### DISCUSSION THREAD

**Discussion by parab-bloom** [prisma/prisma#21338] [Q&A]
### Question

I have multiple env files named (let's say) dev.env, prod.env, stage.env
I am using these files in my app. Now, when I am trying to perform prisma pull it is only supporting `.env` file custom files are not allowed.
To rectify this, I wrote script to perform prisma pull, in this also when I am performing pull for multiple dbs command is separated by && operator. Here, too only first command is getting my environment variables, all other files are not.
Can any one help me with this one.
How can I use custom env files while performing prisma db pull ?

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by janpio** [comment]
https://www.npmjs.com/package/dotenv-cli is a good tool to load any `.env` files when calling Prisma CLI.

**Reply by dejurin** [reply]
For example
```
dotenv -e .env.local -- npx prisma generate
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
